pkg/autodetect/main.go

```diff
@@ -109,16 +108,11 @@ func (b *Background) autoDetectCapabilities() {
 	b.cleanDeployments(ctx)
 }
 
-func (b *Background) availableAPIs(ctx context.Context) (*metav1.APIGroupList, error) {
-	apiList, err := b.dcl.ServerGroups()
-	if err != nil {
-		return nil, err
-	}
-
-	return apiList, nil
+func (b *Background) availableAPIs(_ context.Context) ([]*metav1.APIResourceList, error) {
+	return b.dcl.ServerPreferredResources()
```

Comment:
`b.dcl.ServerGroups()` returns only the API groups and for instance `logging.openshift.io` has multiple kinds: `elasticsearch`, `kibana` ... Hence we should enable ES provisioning only if the `elasticsearch` kind is in the cluster.